src/get-output.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import outputJson from "./output-json.js";
+import outputText from "./output-text.js";
+
+import type {ErrorCode} from "./error-codes.js";
+import type {Options, ILog, JsonItem} from "./types.js";
+
+type OutputFn = (
+    options: Options,
+    log: ILog,
+    jsonItems: Array<JsonItem>,
+    violationFileNames: Array<string>,
+) => ErrorCode;
+
+const getOutput = (options: Options): OutputFn => {
+    const {json} = options;
+
+    return json ? outputJson : outputText;
+};
```

Comment:
Nice :) I know it's super simple right now; I think this will enable us to rework things nicely in the rearchitecture. Thanks for doing this.